src/transformState.js

```diff
@@ -5,7 +5,23 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action in actions) {
+    const removeArr = actions[action].keysToRemove;
+    const actionType = actions[action].type;
+    const extraData = actions[action].extraData;
+
+    if (actionType === 'addProperties') {
+      Object.assign(state, extraData);
+    } else if (actionType === 'removeProperties') {
+      for (const removeProp of removeArr) {
+        delete state[removeProp];
+      }
+    } else if (actionType === 'clear') {
+      for (const key in state) {
+        delete state[key];
+      }
+    }
```

Comment:
avoid such nesting `else if` conditions
This code is hard to read and maintain 
Better to use `switch` here and switch by action type

Reply:
@Bohdan-Maksymiuk done